context/context.go

```diff
@@ -70,6 +71,13 @@ type Context interface {
 
 	// StoreQueryFeedback stores the query feedback.
 	StoreQueryFeedback(feedback interface{})
+
+	// StmtCommit flush all changes by the statement to the underlying transaction.
+	StmtCommit() error
+	// StmtRollback provides statement level rollback.
+	StmtRollback()
+	// GetMutation gets the binlog mutation for current statement.
+	GetMutation(int64) *binlog.TableMutation
```

Comment:
what about GetStmtMutation?

Reply:
I think it can be changed to `StmtGetMutation`, and let all related method name begin with `Stmt` ...